[Bug]:  test_verify_backingstore_uses_rgw pass rate is very low

test_verify_backingstore_uses_rgw pass rate is very low  - is there a marker to add so that we won't have this failed statistics? 

Error: 

/home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/tests/functional/object/mcg/test_provider_client.py:31: Test runs ONLY on cluster with managed service or HCI provider-client platform

on 4.15:
 
https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#OCS/launches/632/34336/1670886/1670987/log

on 4.16: 

https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#OCS/launches/678/34660/1682914/1683013/log

4.17:

https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#OCS/launches/738/34274/1667339/1667445/log

4.18: 

https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#OCS/launches/795/34499/1677900/1677999/log

4.19:

https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#OCS/launches/852/34368/1671721/1671830/log
https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#ocs/launches/852/34728/1685476/1685597/log

mcg_obj_session = <ocs_ci.ocs.resources.mcg.MCG object at 0x7f5231700b50>

    @mcg
    @red_squad
    @runs_on_provider
    @provider_client_ms_platform_required
    @tier1
    @polarion_id("OCS-5415")
    def test_verify_backingstore_uses_rgw(mcg_obj_session):
        """
        Validates whether default MCG backingstore uses rgw endpoint
        """
>       ceph_object_store = OCP(
            kind=constants.CEPHOBJECTSTORE,
            resource_name="ocs-storagecluster-cephobjectstore",
        ).get()

tests/functional/object/mcg/test_provider_client.py:42: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
ocs_ci/ocs/ocp.py:341: in get
    return self.exec_oc_cmd(
ocs_ci/ocs/ocp.py:216: in exec_oc_cmd
    out = run_cmd(
ocs_ci/utility/utils.py:492: in run_cmd
    completed_process = exec_cmd(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

cmd = ['oc', '--kubeconfig', '/home/jenkins/clusters/j010-pcbaie-t1/openshift-cluster-dir/auth/kubeconfig', 'get', 'cephobjectstore', 'ocs-storagecluster-cephobjectstore', ...]
secrets = None, timeout = 600, ignore_error = False, threading_lock = None
silent = False, use_shell = False
cluster_config = <ocs_ci.framework.MultiClusterConfig object at 0x7f528578e820>
lock_timeout = 7200, output_file = None, kwargs = {}
_env = {'BASH_FUNC_which%%': '() {  ( alias;\n eval ${which_declare} ) | /usr/bin/which --tty-only --read-alias --read-functi...D_DISPLAY_NAME': '#10 - Prod ODF Provider Client - BM AI -THALES - tier1', 'BUILD_ID': '10', 'BUILD_NUMBER': '10', ...}
kubeconfig_path = '/home/jenkins/clusters/j010-pcbaie-t1/openshift-cluster-dir/auth/kubeconfig'

    def exec_cmd(
        cmd,
        secrets=None,
        timeout=600,
        ignore_error=False,
        threading_lock=None,
        silent=False,
        use_shell=False,
        cluster_config=None,
        lock_timeout=7200,
        output_file=None,
        **kwargs,
    ):
        """
        Run an arbitrary command locally
    
        If the command is grep and matching pattern is not found, then this function
        returns "command terminated with exit code 1" in stderr.
    
        Args:
            cmd (str): command to run
            secrets (list): A list of secrets to be masked with asterisks
                This kwarg is popped in order to not interfere with
                subprocess.run(``**kwargs``)
            timeout (int): Timeout for the command, defaults to 600 seconds.
            ignore_error (bool): True if ignore non zero return code and do not
                raise the exception.
            threading_lock (threading.RLock): threading.RLock object that is used
                for handling concurrent oc commands
            silent (bool): If True will silent errors from the server, default false
            use_shell (bool): If True will pass the cmd without splitting
            cluster_config (MultiClusterConfig): In case of multicluster environment this object
                    will be non-null
            lock_timeout (int): maximum timeout to wait for lock to prevent deadlocks (default 2 hours)
            output_file (str): path where to write output of stderr from command - apply only when silent mode is True
    
        Raises:
            CommandFailed: In case the command execution fails
    
        Returns:
            (CompletedProcess) A CompletedProcess object of the command that was executed
            CompletedProcess attributes:
            args: The list or str args passed to run().
            returncode (str): The exit code of the process, negative for signals.
            stdout     (str): The standard output (None if not captured).
            stderr     (str): The standard error (None if not captured).
    
        """
        _env = kwargs.pop("env", os.environ.copy())
        kubeconfig_path = config.RUN.get("kubeconfig")
        if kubeconfig_path:
            _env["KUBECONFIG"] = kubeconfig_path
        if cluster_config:
            kubeconfig_path = cluster_config.RUN.get("kubeconfig")
            if kubeconfig_path:
                _env["KUBECONFIG"] = cluster_config.RUN.get("kubeconfig")
        if isinstance(cmd, str) and not kwargs.get("shell"):
            cmd = shlex.split(cmd)
        if (
            kubeconfig_path
            and cmd[0] == "oc"
            and "--kubeconfig" not in cmd
            and "mirror" not in cmd
        ):
            kube_index = 1
            # check if we have an oc plugin in the command
            plugin_list = "oc plugin list"
            cp = subprocess.run(
                shlex.split(plugin_list),
                stdout=subprocess.PIPE,
                stderr=subprocess.PIPE,
            )
            subcmd = cmd[1].split("-")
            if len(subcmd) > 1:
                subcmd = "_".join(subcmd)
            if not isinstance(subcmd, str) and isinstance(subcmd, list):
                subcmd = str(subcmd[0])
    
            for l in cp.stdout.decode().splitlines():
                if subcmd in l:
                    # If oc cmdline has plugin name then we need to push the
                    # --kubeconfig to next index
                    kube_index = 2
                    log.info(f"Found oc plugin {subcmd}")
            cmd = list_insert_at_position(cmd, kube_index, ["--kubeconfig"])
            cmd = list_insert_at_position(cmd, kube_index + 1, [kubeconfig_path])
        try:
            if kwargs.get("shell"):
                masked_cmd = mask_secrets(cmd, secrets)
            else:
                masked_cmd = shlex.join(mask_secrets(cmd, secrets))
            log.info(f"Executing command: {masked_cmd}")
            if threading_lock and cmd[0] == "oc":
                threading_lock.acquire(timeout=lock_timeout)
            completed_process = subprocess.run(
                cmd,
                stdout=subprocess.PIPE,
                stderr=subprocess.PIPE,
                stdin=subprocess.PIPE,
                timeout=timeout,
                env=_env,
                **kwargs,
            )
        finally:
            if threading_lock and cmd[0] == "oc":
                threading_lock.release()
        masked_stdout = mask_secrets(completed_process.stdout.decode(), secrets)
        if len(completed_process.stdout) > 0:
            log.debug(f"Command stdout: {masked_stdout}")
        else:
            log.debug("Command stdout is empty")
    
        masked_stderr = mask_secrets(completed_process.stderr.decode(), secrets)
        if len(completed_process.stderr) > 0:
            if not silent:
                log.warning(f"Command stderr: {masked_stderr}")
            else:
                if output_file:
                    with open(output_file, "a") as out_fd:
                        out_fd.write(masked_stderr)
        else:
            if not silent:
                log.debug("Command stderr is empty")
        log.debug(f"Command return code: {completed_process.returncode}")
        if completed_process.returncode and not ignore_error:
            masked_stderr = bin_xml_escape(filter_out_emojis(masked_stderr))
            if (
                "grep" in masked_cmd
                and b"command terminated with exit code 1" in completed_process.stderr
            ):
                log.info(f"No results found for grep command: {masked_cmd}")
            else:
>               raise CommandFailed(
                    f"Error during execution of command: {masked_cmd}."
                    f"\nError is {masked_stderr}"
                )
E               ocs_ci.ocs.exceptions.CommandFailed: Error during execution of command: oc --kubeconfig /home/jenkins/clusters/j010-pcbaie-t1/openshift-cluster-dir/auth/kubeconfig get cephobjectstore ocs-storagecluster-cephobjectstore -o yaml.
E               Error is Error from server (NotFound): cephobjectstores.ceph.rook.io "ocs-storagecluster-cephobjectstore" not found

--- a/tests/functional/object/mcg/test_provider_client.py
+++ b/tests/functional/object/mcg/test_provider_client.py
@@ -8,6 +8,7 @@ from ocs_ci.framework.pytest_customization.marks import (
     run_on_all_clients_push_missing_configs,
     runs_on_provider,
     mcg,
+    pc_or_ms_provider_required,
     provider_client_ms_platform_required,
 )
 from ocs_ci.ocs import constants
@@ -32,7 +33,7 @@ def return_to_original_context():
 @mcg
 @red_squad
 @runs_on_provider
-@provider_client_ms_platform_required
+@pc_or_ms_provider_required
 @tier1
 @polarion_id("OCS-5415")
 def test_verify_backingstore_uses_rgw(mcg_obj_session):

